source/sdk/lang/Iterators.ooc

```diff
@@ -4,101 +4,100 @@ import structs/[List, ArrayList] /* for Iterable<T> toArrayList */
  * iterators
  */
 Iterable: abstract class <T> {
-
-    iterator: abstract func -> Iterator<T>
-
-    /** Return the contents of the iterable as an ArrayList. */
-    toList: func -> ArrayList<T> {
-        result := ArrayList<T> new()
-        for(elem: T in this) {
-            result add(elem)
-        }
-        result
-    }
-
-    reduce: func (f: Func (T, T) -> T) -> T {
-        iter := iterator()
-        acc := f(iter next(), iter next())
-        while(iter hasNext?()) acc = f(acc, iter next())
-        acc
-    }
-
-    each: func (f: Func (T)) {
-        for(elem in this) {
-            f(elem)
-        }
-    }
-
-    // Return false to break
-    eachUntil: func (f: Func (T) -> Bool) {
-        for(elem in this) {
-            if(!f(elem)) break
-        }
-    }
-    
-    each: func ~withIndex (f: Func (T, Int)) {
-        index := 0
-        for(elem in this) {
-            f(elem, index)
-            index += 1
-        }
-    }
-
+	iterator: abstract func -> Iterator<T>
+	/** Return the contents of the iterable as an ArrayList. */
+	toList: func -> ArrayList<T> {
+		result := ArrayList<T> new()
+		for(elem: T in this)
+			result add(elem)
+		result
+	}
+	reduce: func (f: Func (T, T) -> T) -> T {
+		iter := iterator()
+		acc := f(iter next(), iter next())
+		while(iter hasNext?())
+			acc = f(acc, iter next())
+		acc
+	}
+	each: func (f: Func (T)) {
+		for(elem in this)
+			f(elem)
+	}
+	// Return false to break
+	eachUntil: func (f: Func (T) -> Bool) {
+		for(elem in this)
+			if(!f(elem)) break
+	}
+	each: func ~withIndex (f: Func (T, Int)) {
+		index := 0
+		for(elem in this) {
+			f(elem, index)
+			index += 1
+		}
+	}
```

Comment:
I did it because I didn't want to have mixed styles in the file, by editor settings or by mistake. We're going to add some more stuff to that file later on.